features/inventory/checkout_inventory/holding_inventory_units_during_checkout_process.feature

```diff
@@ -0,0 +1,31 @@
+@checkout_inventory
+Feature: Holding inventory units during checkout
+    In order to be sure that products I buy were not bought by another customer
+    As an Administrator
+    I want to buy selected products
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Iron Maiden T-Shirt"
+        And "Iron Maiden T-Shirt" product is tracked by the inventory
+        And there are 5 units of product "Iron Maiden T-Shirt" available in the inventory
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a user "sylius@example.com" identified by "sylius"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Holding inventory units
+        Given this user has added 3 products "Iron Maiden T-Shirt" to the cart
+        And this user bought those products
+        When I view all variants of the product "Iron Maiden T-Shirt"
```

Comment:
I think that `When I view variants of product "Iron Maiden T-Shirt"` will be more appropriate step.